lib/reek/cli/command/todo_list_command.rb

```diff
@@ -12,17 +12,19 @@ module Command
       # file that can serve as a todo list.
       #
       class TodoListCommand < BaseCommand
-        FILE_NAME = '.todo.reek'
+        DEFAULT_CONFIGURATION_FILE_NAME = Configuration::ConfigurationFileFinder::DEFAULT_FILE_NAME
+        DETECTOR_KEY = Configuration::AppConfiguration::DETECTORS_KEY
+        HEADER = "# Auto generated by Reeks --todo flag\n"
+        EXISTING_FILE_MESSAGE = "\nExisting '#{DEFAULT_CONFIGURATION_FILE_NAME}' detected - aborting.\n"
+        DO_NOTHING_MESSAGE = "\nNo smells found - nothing to do, exiting.\n"
 
         def execute
           if smells.empty?
-            puts "\n'.todo.reek' not generated because "\
-                    'there were no smells found!'
+            puts DO_NOTHING_MESSAGE
```

Comment:
Let's rename this to `NO_SMELLS_FOUND_MESSAGE` to match `EXISTING_FILE_MESSAGE`, since both messages indicate a situation where nothing is done.